P3527R0.md

```diff
@@ -65,110 +59,131 @@ proposal.  It is here to describe the relevant language part of the pattern
 matching design, since that language design informs the library design in this
 paper.
 
+To match based on the currently-engaged value of a variant-like object `v` of
+type `V`, the pattern matching language feature uses `std::variant_size<V>`,
+`std::variant_alternative<V>`, `std::get<T>(v)`, and `v.index()`.  The
+`std::get` overloads used are the ones that take a type template parameter,
+not the ones that take a `size_t` NTTP.
```

Comment:
It is actually `get<I>` that is used. The usage is basically like:
```
switch (v.index()) {
  case Idx: {
    std::variant_alternative_t<Idx, V>& __temp = get<Idx>(v);
  // for Idx 0 ... std::variant_size_v<V> - 1
  }
}
```
I wouldn't say `std::get`, since it's an ADL-only `get` call.

Reply:
Yeah, that's what I meant to write.  I just flipped it the first time, and then repeated the error after that.